include/grpcpp/impl/codegen/async_unary_call.h

```diff
@@ -375,7 +374,7 @@ class ServerAsyncResponseWriter final
   /// gRPC doesn't take ownership or a reference to status, so it is safe to
   /// deallocate them once the Finish operation is complete (i.e. a result
   /// arrives in the completion queue).
-  void FinishWithError(const ::grpc::Status& status, void* tag) {
+  void FinishWithError(const grpc::Status& status, void* tag) {
```

Comment:
what's the chance of namespace collisions for all these changes in header files?

Reply:
Discussed offline, it would be a challenge to make this change problematic. If you can come up with a counterexample, please share!